src/test/resources/features/user/smokeUserScenarios.feature

```diff
@@ -0,0 +1,26 @@
+@SMOKE
+Feature: Smoke testing the users endpoints
+
+  Background: Creating a user
+    When I create a new user with email uniqueol@email.com, first name Dan, last name Smith and gender male
```

Comment:
`{
    "_meta": {
        "success": false,
        "code": 422,
        "message": "Data validation failed. Please check the response body for detailed error messages.",
        "rateLimit": {
            "limit": 60,
            "remaining": 59,
            "reset": 1
        }
    },
    "result": [
        {
            "field": "email",
            "message": "Email \"email9hl@myemail.com\" has already been taken."
        }
    ]
}`
This happened because the tearDown method wasn't executed, because the test before it failed. When we discussed I said a better approach will be to create a step like "When I create a new user with random email, etc". Other option is to fix the tearDown method.